Lib/test/test_tools/test_pathfix.py

```diff
@@ -0,0 +1,48 @@
+import unittest, os, subprocess
+from test.test_tools import import_tool, scriptsdir
+import sys
+from test import support
+
+
+class TestPathfixFunctional(unittest.TestCase):
+    script = os.path.join(scriptsdir, 'pathfix.py')
+
+    def setUp(self):
+        self.temp_file = support.TESTFN
+        self.addCleanup(support.unlink, support.TESTFN)
+
+    def pathfix(self, shebang, pathfix_flags):
+        with open(self.temp_file, 'w') as f:
```

Comment:
I would prefer to specify the encoding: pass encoding="utf8" to this open, and the other one below.